testcrate/build.rs

```diff
@@ -0,0 +1,26 @@
+use std::{env, fs};
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+
+    let include = env::var("DEP_SODIUM_INCLUDE").unwrap();
+    let header = fs::read_dir(include)
+        .unwrap()
+        .filter_map(Result::ok)
+        .find(|entry| entry.file_name() == "sodium.h");
+    assert!(
+        header.is_some(),
+        "sodium.h not found in DEP_SODIUM_INCLUDE dir"
+    );
+
+    let lib = env::var("DEP_SODIUM_LIB").unwrap();
+    let compiled_lib = fs::read_dir(lib)
+        .unwrap()
+        .filter_map(Result::ok)
+        .find(|e| e.file_name() == "libsodium.a");
```

Comment:
not only libsodium.a but .dll or .lib too.

Reply:
Yes, I was waiting for the CI to fail so that I can see which file is needed in the windows build.